src/libmain/progress-bar.cc

```diff
@@ -17,20 +16,32 @@ static std::string getS(const std::vector<Logger::Field> & fields, size_t n)
     return fields[n].s;
 }
 
+static std::string storePathToName(std::string path)
```

Comment:
Maybe use the original version? `std::string_view` is cheaper than `std::string`.